src/ofp_arp.c

```diff
@@ -708,7 +721,9 @@ void ofp_arp_show_table(int fd)
 	for (i = 1; i < NUM_ARPS; ++i) {
 		entry = &shm->arp.entries[i];
 
-		if (entry->flags.is_complete)
+		if (entry->flags.is_complete ||
+		    (!entry->flags.is_complete &&
```

Comment:
No need to repeat this clause after the first one.

Reply:
OK